js/templates/aliases-list.html

```diff
@@ -0,0 +1,18 @@
+{{#if aliases}}
+<table class="grid">
+	<tbody>
+	{{#each aliases}}
+	<tr id="{{ id }}">
+		<td>
+			{{ alias }}
+		</td>
+		<td>
+			{{ name }}
+		</td>
+		<td> <button type="submit" value="{{ id }}" class="icon-delete"></button>
+		</td>
+	</tr>
+	{{/each}}
+	</tbody>
+</table>
```

Comment:
AFAIK a CompositeView would be handy here. You could leave the template as it is now, but specify the ``tbody`` element as [childViewContainer](https://github.com/marionettejs/backbone.marionette/blob/master/docs/marionette.compositeview.md#compositeviews-childviewcontainer)